### DISCUSSION TITLE
Maa combat can't start

### DISCUSSION THREAD

**Discussion by EfJun1** [MaaAssistantArknights/MaaAssistantArknights#8443] [Q&A] [answered]
![MAA_v5_hzfNIVozpm](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/130586677/96ac8a85-2666-4ebc-95b8-6b867d968777)
Hi, I had this error today and I can't fix it. Everything else is fine but combat just can't start.

**Comment by status102** [comment, marked as answer]
submit an issue with log
it may be caused by changed ui i guess. If that, wait for update. A announcement has been published.

**Reply by EfJun1** [reply]
[gui.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/14467951/gui.log)
This should be the log, let me know what else I can submit.

**Reply by status102** [reply]
asst.log is also necessary. and recommend to submit an issue.
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose
Because there is not automated classification and developers usually use issue.

**Reply by EfJun1** [reply]
Noted, submitted. Thank you very much for your help.
